common/api/summary/frontend-authorization-client.exports.csv

```diff
@@ -8,7 +8,6 @@ beta;BrowserAuthorizationClientConfiguration
 internal;BrowserAuthorizationClientRedirectState
 public;BrowserAuthorizationClientRequestOptions
 beta;BrowserAuthorizationLogger 
-beta;FrontendAuthorizationClient 
```

Comment:
Can you delete this file, the corresponding api.md file and the all of the changelogs in `common/changes/@bentley/frontend-authorization-client`?

Reply:
Removed.